### PR TITLE
Add Makefile

### PR DESCRIPTION
**Pull request by AurelienGasser**
A few useful commands, to avoid having to search them in the README.﻿

### CODE REVIEW

**Comment by inalgnu** [comment]
@AurelienGasser wouldn't be better to have the Makefile located in the root project directory?

**Comment by AurelienGasser** [comment]
@inalgnu I was on the fence myself. Let me move it.

**Comment by AurelienGasser** [comment]
@inalgnu done

**Comment by AurelienGasser** [comment]
@inalgnu I made a few additional changes based on the bugs I encountered working on https://github.com/SubstraFoundation/substra-backend/pull/219. 

Would you mind reviewing again?

File: backend/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.PHONY: exception_map
+exception_map:
+	# Fully rebuild the reference file, because upading the reference file
+	# in-place occasionally leads to failed unit tests.
+	rm substrapp/tasks/exceptions.json
+
+	# Use python3.7 so that the exception map is compatible with the python
+	# version used in Travis unit test builds.
+	python3.7 substrapp/tasks/exception_handler.py
```
Comment (inalgnu) [inline backend/Makefile:17]: missing EOF

**Pull request merged**